Library/Homebrew/dev-cmd/man.rb

```diff
@@ -56,11 +56,49 @@ def path_glob_commands(glob)
     end.reject { |s| s.strip.empty? || s.include?("@hide_from_man_page") }
   end
 
+  def path_glob_commands_from_DSL(glob)
```

Comment:
This method is hard to follow. I think it should be split into more, smaller methods to signal intent.